templates/var/backups/openldap/config.ldif

```diff
@@ -26,12 +27,16 @@ olcServerID: {{ ldaptoolbox_openldap_olcServerID }}
 olcSockbufMaxIncoming: 262143
 olcSockbufMaxIncomingAuth: 16777215
 olcThreads: 16
+{% if ldaptoolbox_openldap_olcTLSCACertificateFile %}
```

Comment:
This is modifing the current behaviour of `olcTLSProtocolMin`

I have fixed this in rpm branch.